src/isa/instr.rs

```diff
@@ -883,6 +883,16 @@ pub enum DigestOp {
         /** Index of string register */ RegS,
         /** Index of `r512` register to save result to */ Reg16,
     ),
+
+    /// Computes BLAKE3 hash value
+    ///
+    /// Sets `st0` to `false` and destination register to `None` if the source register does not
+    /// contain a value
+    #[display("Blake3    {0},r256{1}")]
```

Comment:
can you please change mnemonic so it looks more in line with the rest
```suggestion
    #[display("blake3  {0},r256{1}")]
```